### ISSUE TITLE
protein prep is currently removing crystal waters

### ISSUE THREAD

**Issue by JenkeScheen** [choderalab/asapdiscovery#609]
we need to retain crystallographic waters throughout our prep workflows so that we make sure that these are used for docking, viz and FECs.

**Comment by apayne97** [comment]
yes we used to be able to do that, shouldn't be too hard to add back
